hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/conf/YarnConfiguration.java

```diff
@@ -2158,6 +2158,10 @@ public static boolean isAclEnabled(Configuration conf) {
       "org.apache.hadoop.yarn.server.nodemanager.containermanager" +
       ".linux.runtime.runc.HdfsManifestToResourcesPlugin";
 
+  /** The runc container transformer plugin to use.  If empty, no
+   * transformations will be applied. */
+  public static final String NM_RUNC_CONTAINER_TRANSFORMER_PLUGIN = "";
```

Comment:
I'm pretty sure this is being used as the conf key, not the value, so an empty string is wrong here.

Reply:
Yes, good catch.  This was updated to match the other runC configurations.